recipe/recipe.yaml

```diff
@@ -0,0 +1,124 @@
+# ensure mpi is defined (needed for conda-smithy recipe-lint)
+# prioritize nompi variant via build number
+schema_version: 1
+
+context:
+  name: openpmd-api
+  version: 0.16.1
+  build: 0
+  mpi: ${{ mpi or "nompi" }}  # is this still needed?
+  mpi_prefix: ${{ "nompi" if mpi == "nompi" else "mpi_" + mpi }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/openPMD/openPMD-api/archive/${{ version }}.tar.gz
+  sha256: a029a1779351949f41c1f36d0e75c698e59c5d284f080d5e4c2b8650779d2d58
+
+build:
+  # add build string so packages can depend on
+  # mpi or nompi variants
+  # dependencies:
+  # `pkg * mpi_mpich_*` for mpich
+  # `pkg * mpi_*` for any mpi
+  number: ${{ build | int + 100 if mpi == "nompi" else build }}
+  string: ${{ mpi_prefix }}_py${{ py }}h${{ PKG_HASH }}_${{ build }}
```

Comment:
Oh, somehow the created files now lak the `py` version and `PKG_HASH`, e.g., now read: `linux-64/openpmd-api-0.16.1-mpi_mpich_pyh_0.conda`
https://anaconda.org/conda-forge/openpmd-api/files

@wolfv did I do something wrong here?

Reply:
Attempting a fix in #119 based on a search on conda-forge repos that also set `string`

Reply:
```suggestion
  string: ${{ mpi_prefix }}_py${{ python | version_to_buildstring }}h${{ hash }}_${{ build }}
```